docker-compose-dev.yaml

```diff
@@ -196,6 +198,76 @@ services:
     ports:
       - 8083:3000
 
+  badgerdoc-zookeeper:
+    image: wurstmeister/zookeeper
+    container_name: badgerdoc-zookeeper
+    ports:
+      - "2181:2181"
+    networks:
+      - badgerdoc
+
+  badgerdoc-kafka:
+    image: wurstmeister/kafka
+    container_name: badgerdoc-kafka
+    ports:
+      - "9092:9092"
+    environment:
+      KAFKA_ADVERTISED_HOST_NAME: badgerdoc-kafka
+      KAFKA_ZOOKEEPER_CONNECT: badgerdoc-zookeeper:2181
+      KAFKA_CREATE_TOPICS: 'scheduler:1:1'
+    networks:
+      - badgerdoc
+    depends_on:
+      - badgerdoc-zookeeper
+
+  badgerdoc-pipelines:
+    image: badgerdoc_pipelines
+    container_name: pipelines
+    env_file:
+      - .env
+    environment:
+      - ROOT_PATH=/pipelines
+    networks:
+      - badgerdoc
+    command: bash -c "./wait-for-it.sh -t 0 badgerdoc-kafka:9092 && alembic upgrade head && uvicorn pipelines.app:app --host 0.0.0.0 --port 8080 --reload"
```

Comment:
Do we need this? `pipelines` Dockerfile already has an entry point
https://github.com/epam/badgerdoc/blob/main/pipelines/Dockerfile#L18

Reply:
Yes, we need. Dockerfile doesn't contain running of ./wait-for-it.sh